app/views/casework/decisions/comments/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<% title t '.title' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body-l govuk-!-margin-bottom-2">
+      <%= @crime_application.reference %>
+    </p>
+
+    <h2 class="govuk-heading-xl govuk-!-margin-bottom-4">
+      <%= @crime_application.applicant.full_name %>
+    </h2>
+
+    <%= govuk_error_summary(@form_object) %>
+  </div>
+
+  <div class="govuk-grid-column-full">
+    <%= render DecisionComponent.new(decision: @decision) %>
+  </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @form_object, url: crime_application_decision_comment_path, method: :put do |f| %>
+
+      <%= f.govuk_radio_buttons_fieldset :comment_required do %>
+        <%= f.govuk_radio_button :comment_required, 'true' do %>
+          <%= f.govuk_text_area :comment %>
+        <% end %>
+
+        <%= f.govuk_radio_button :comment_required, 'false' %>
+      <% end %>
+
+      <%= f.govuk_submit %>
+    <% end %>
```

Comment:
Was testing this and if you add a comment then select no, it saves the comment

Reply:
Good spot. Thank you, will fix 😄